packages/contracts-versions/commit_hashes.json

```diff
@@ -0,0 +1,16 @@
+{
+  "versions": [
+    {
+      "name": "v0.7.0_alpha",
+      "commit": "9b912728c315f983af8c4bfd1f8d07021996563f"
```

Comment:
As suggested in our Tech Debt inbox (
https://www.notion.so/aragonorg/Add-git-version-tags-to-the-osx-repo-d489455eaeb54e499268d34add73a844?pvs=4), it would be good to use git tags in our repository for OSx. 

The field "tag" would then be an alternative for `"name"` and `"commit"`.
The script could then just do `git checkout tags/v1.0.0`.

In the next iteration, we could also think about creating a workflow to automatically create a release and adding the files to the `contracts-version` npm package.

I think using git tags and publishing tagged commits automatically to the `contracts-version` npm package could be favorable over doing this manually. Ping @chilcano and @mathewmeconry for their opinion.

Reply:
Let's create a follow-up task for this.  Resolving it for now